token/core/zkatdlog/crypto/transfer/sender.go

```diff
@@ -123,7 +122,7 @@ type Action struct {
 	// Inputs specify the identifiers in of the tokens to be spent
 	Inputs []*token2.ID
```

Comment:
Since we are talking about an external interface, I think it would be cleaner to have a type
```
type Token struct {
token.ID
token.Token
}
```